### PR TITLE
Add iceberg documentation

### PR DESCRIPTION
**Pull request by AyushSehrawat**

### CODE REVIEW

**Review by dreulavelle — APPROVED** [review]

**Review by dreulavelle — CHANGES_REQUESTED** [review]
Dont forget to change all names to Riven

**Comment by AyushSehrawat** [comment]
Will review today 👍🏻

**Review by dreulavelle — APPROVED** [review]

File: docs/riven/index.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+
+Rclone is mounted to /riven/vfs on your host machine -> settings should have: "host_mount": "/riven/vfs"
+
+Plex container volume configuration for rclone mount is "/riven/vfs:/media/vfs" -> settings should have: "container_mount": "/media/vfs"
+
+Plex libraries you want to add to sections: movies -> /media/library/movies, shows -> /media/library/shows
+
+--- -->
+
+## Development
```
Comment (dreulavelle) [inline docs/riven/index.md:302]: We shouldn't put development stuff here in this wiki. That should stay with the Riven repo

File: docs/riven/index.md [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+
+```sh
+cd frontend
+npm install
+npm run dev
+```
+
+---
+
+## Contributing
```
Comment (dreulavelle) [inline docs/riven/index.md:340]: We can shorten a contributing section probably just to point people to the repo? Or get rid of it altogether

File: docs/riven/index.md [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+
+<h1 style="color: red">IS THIS SECTION STILL NEEDED? OR SHOULD THIS BE EXPLAINED ABOVE?</h1>
+
+"host_mount" should point to your rclone mount that has your torrents on your host, if you are using native webdav set webdav-url to "https://dav.real-debrid.com/torrents"
+
+"container_mount" should point to the location of the mount in plex container
+
+#### Example:
+
+Rclone is mounted to /riven/vfs on your host machine -> settings should have: "host_mount": "/riven/vfs"
```
Comment (dreulavelle) [inline docs/riven/index.md:294]: All the rclone/symlink stuff is incorrect

File: docs/riven/index.md (hunk 1 above)
Comment (AyushSehrawat) [reply docs/riven/index.md:302]: Okay!

File: docs/riven/index.md (hunk 2 above)
Comment (AyushSehrawat) [reply docs/riven/index.md:340]: Yea sure, that works too

File: docs/riven/index.md (hunk 3 above)
Comment (AyushSehrawat) [reply docs/riven/index.md:294]: Will see and edit accordingly

Comment (AyushSehrawat) [reply docs/riven/index.md:294]: @davidemarcoli

**Comment by dreulavelle** [comment]
I'm thinking to merge this pr so that I can open the project up in Obsidian and start working on the Riven docs. 

Is that alright? @davidemarcoli @AyushSehrawat

**Comment by davidemarcoli** [comment]
@dreulavelle For me that's fine. I've not rewritten everything, so there may be some old stuff from the readme (for example the symlink stuff).

**Comment by AyushSehrawat** [comment]
> I'm thinking to merge this pr so that I can open the project up in Obsidian and start working on the Riven docs.
> 
> Is that alright? @davidemarcoli @AyushSehrawat

You can do npm run docs:dev too

**Pull request merged**